code/obj/machinery/phone/phone.dm

```diff
@@ -42,7 +42,7 @@ TYPEINFO(/obj/machinery/phone)
 			if(istype(location,/area/station/security))
 				stripe_color = "#ff0000"
 				phone_category = "security"
-			else if(istype(location,/area/station/bridge))
+			else if(istype(location,/area/station/bridge) || isAI(src.loc))
```

Comment:
This should go at the top of the checks otherwise AI phones will get categorized under security if the AI core is in security